### PR TITLE
[ES] Update `Language Surveillance Division`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Language_Surveillance_Division/es.md [hunk 1]
```diff
@@ -10,22 +10,22 @@ tags:
   - ops del chat
 ---
 
-# División de Vigilancia Lingüística
+# Language Surveillance Division
```
Comment (UnknownOsu) [inline wiki/People/Language_Surveillance_Division/es.md:13]:
```suggestion
# División de Vigilancia Lingüística
```

Comment (UnknownOsu) [reply wiki/People/Language_Surveillance_Division/es.md:13]: Creo que queda mejor en español

File: wiki/People/Language_Surveillance_Division/es.md [hunk 2]
```diff
@@ -10,22 +10,22 @@ tags:
...
+*Para conocer el equipo moderno de moderadores, véase: [Global Moderation Team](/wiki/People/Global_Moderation_Team).*
 
-La **División de Vigilancia Lingüística** (coloquialmente conocida como *LSD*) era un grupo de usuarios responsable de la moderación de comunidades de idiomas específicos dentro de osu!. Cada uno de sus miembros tenía un área de enfoque dedicada: un canal de idioma en el que estaban presentes principalmente y las secciones respectivas del foro que podían administrar (como lugares como [Ayuda](https://osu.ppy.sh/community/forums/5) o [Fuera de tema](https://osu.ppy.sh/community/forums/52)).
+La **Language Surveillance Division** (coloquialmente conocida como *LSD*) era un grupo de usuarios responsable de la moderación de comunidades de idiomas específicos dentro de osu!. Cada uno de sus miembros tenía un área de enfoque dedicada: un canal de idioma en el que estaban presentes principalmente y las secciones respectivas del foro que podían administrar (como lugares como [Help](https://osu.ppy.sh/community/forums/5) o [Off-Topic](https://osu.ppy.sh/community/forums/52)).
```
Comment (UnknownOsu) [inline wiki/People/Language_Surveillance_Division/es.md:17]:
```suggestion
La **División de Vigilancia Lingüística** (coloquialmente conocida como *LSD*) era un grupo de usuarios responsable de la moderación de comunidades de idiomas específicos dentro de osu!. Cada uno de sus miembros tenía un área de enfoque dedicada: un canal de idioma en el que estaban presentes principalmente y las secciones respectivas del foro que podían administrar (como lugares como [Help](https://osu.ppy.sh/community/forums/5) u [Off-Topic](https://osu.ppy.sh/community/forums/52)).
```

File: wiki/People/Language_Surveillance_Division/es.md [hunk 3]
```diff
@@ -10,22 +10,22 @@ tags:
...
+Los miembros de la LSD fueron seleccionados personalmente por el osu! team entre la comunidad para actuar como moderadores en comunidades y subforos de menor tráfico. A diferencia de los moderadores globales, no tenían características como apodos de colores o títulos de foros personalizados. En [IRC](/wiki/Community/Internet_Relay_Chat), sin embargo, eran [operadores](/wiki/Community/Internet_Relay_Chat#¿por-qué-algunos-nombres-de-usuario-tienen-prefijos-con-diferentes-signos?) y podían aplicar [silencios](/wiki/Silence).
 
-El LSD también sirvió como un grupo intermediario para miembros prometedores pendientes de admisión en el [Equipo de Moderación Global](/wiki/People/Global_Moderation_Team) en ocasiones, ya que los dos grupos generalmente compartían un objetivo común y conjuntos de habilidades similares.
+El LSD también sirvió como un grupo intermediario para miembros prometedores pendientes de admisión en el [Global Moderation Team](/wiki/People/Global_Moderation_Team) en ocasiones, ya que los dos grupos generalmente compartían un objetivo común y conjuntos de habilidades similares.
```
Comment (UnknownOsu) [inline wiki/People/Language_Surveillance_Division/es.md:23]:
```suggestion
La LSD también sirvió como un grupo intermediario para miembros prometedores pendientes de admisión en el [Global Moderation Team](/wiki/People/Global_Moderation_Team) en ocasiones, ya que los dos grupos generalmente compartían un objetivo común y conjuntos de habilidades similares.
```

File: wiki/People/Language_Surveillance_Division/es.md [hunk 4]
```diff
@@ -10,22 +10,22 @@ tags:
...
 
 ## Reestructuración
 
 ![](img/chat-moderator-profile.png "Página de perfil de un moderador del chat en 2013")
 
-Después de una [reestructuración del personal](https://osu.ppy.sh/community/forums/topics/123510) en 2013, la División de Vigilancia Lingüística pasó a llamarse **Moderadores del Chat** y recibió su propia lista de miembros.
+Después de una [reestructuración del personal](https://osu.ppy.sh/community/forums/topics/123510) en 2013, la Language Surveillance Division pasó a llamarse **Chat Moderators** y recibió su propia lista de miembros.
```
Comment (UnknownOsu) [inline wiki/People/Language_Surveillance_Division/es.md:29]:
```suggestion
Después de una [reestructuración del personal](https://osu.ppy.sh/community/forums/topics/123510) en 2013, la División de Vigilancia Lingüística pasó a llamarse **Chat Moderators** y recibió su propia lista de miembros.
```

File: wiki/People/Language_Surveillance_Division/es.md [hunk 5]
```diff
@@ -10,22 +10,22 @@ tags:
...
 
-El [20 de agosto de 2015](/wiki/People/Staff_log/2015#agosto), los equipos de Moderadores del Chat y Moderación Global [se fusionaron](https://osu.ppy.sh/community/forums/posts/4435972), dando efectivamente a todos los moderadores apodos rojos distintivos en el chat y poder de administración sobre todo el foro.
+El [20 de agosto de 2015](/wiki/People/Staff_log/2015#agosto), los equipos Chat Moderators y Global Moderation [se fusionaron](https://osu.ppy.sh/community/forums/posts/4435972), dando efectivamente a todos los moderadores apodos rojos distintivos en el chat y poder de administración sobre todo el foro.
```
Comment (UnknownOsu) [inline wiki/People/Language_Surveillance_Division/es.md:31]:
```suggestion
El [20 de agosto de 2015](/wiki/People/Staff_log/2015#agosto), los equipos de Chat Moderators y Global Moderation [se fusionaron](https://osu.ppy.sh/community/forums/posts/4435972), dando efectivamente a todos los moderadores apodos rojos distintivos en el chat y poder de administración sobre todo el foro.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/People/Language_Surveillance_Division/es.md [hunk 6]
```diff
@@ -12,20 +12,20 @@ tags:
...
+*Para conocer el equipo moderno de moderadores, véase: [Global Moderation Team](/wiki/People/Global_Moderation_Team).*
 
-La **División de Vigilancia Lingüística** (coloquialmente conocida como *LSD*) era un grupo de usuarios responsable de la moderación de comunidades de idiomas específicos dentro de osu!. Cada uno de sus miembros tenía un área de enfoque dedicada: un canal de idioma en el que estaban presentes principalmente y las secciones respectivas del foro que podían administrar (como lugares como [Ayuda](https://osu.ppy.sh/community/forums/5) o [Fuera de tema](https://osu.ppy.sh/community/forums/52)).
+La **División de Vigilancia Lingüística** (coloquialmente conocida como *LSD*) era un grupo de usuarios responsable de la moderación de comunidades de idiomas específicos dentro de osu!. Cada uno de sus miembros tenía un área de enfoque dedicada: un canal de idioma en el que estaban presentes principalmente y las secciones respectivas del foro que podían administrar (como lugares como [Help](https://osu.ppy.sh/community/forums/5) u [Off-Topic](https://osu.ppy.sh/community/forums/52)).
```
Comment (UnknownOsu) [inline wiki/People/Language_Surveillance_Division/es.md:17]:
```suggestion
La **División de Vigilancia Lingüística** (coloquialmente conocida como *LSD*) era un grupo de usuarios responsable de la moderación de comunidades de idiomas específicos dentro de osu!. Cada uno de sus miembros tenía un área de enfoque dedicada: un canal de idioma en el que estaban presentes principalmente y las secciones respectivas del foro que podían administrar (lugares como [Help](https://osu.ppy.sh/community/forums/5) u [Off-Topic](https://osu.ppy.sh/community/forums/52)).
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Review by UnknownOsu — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**